rust/README.axum.md

```diff
@@ -0,0 +1,566 @@
+    </a>{% if codecov %}
+    <a href="https://codecov.io/gh/{{github-name}}/{{repo-name}}">
+      <img src="https://codecov.io/gh/{{github-name}}/{{repo-name}}/branch/main/graph/badge.svg?token=SOMETOKEN" alt="Code Coverage"/>
+    </a>{% endif %}{% if github_actions %}
+    <a href="https://github.com/{{github-name}}/{{repo-name}}/actions?query=">
+      <img src="https://github.com/{{github-name}}/{{repo-name}}/actions/workflows/tests_and_checks.yml/badge.svg" alt="Build Status">
+    </a>{% endif %}{% if license == "Apache" %}
+    <a href="https://github.com/{{github-name}}/{{repo-name}}/blob/main/LICENSE">
+      <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="License">
+    </a>{% elsif license == "MIT" %}
+    <a href="https://github.com/{{github-name}}/{{repo-name}}/blob/main/LICENSE">
+      <img src="https://img.shields.io/badge/License-MIT-blue.svg" alt="License">
+    </a>{% else %}
+    <a href="https://github.com/{{github-name}}/{{repo-name}}/blob/main/LICENSE-APACHE">
+      <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="License-Apache">
+    </a>
+    <a href="https://github.com/{{github-name}}/{{repo-name}}/blob/main/LICENSE-MIT">
+      <img src="https://img.shields.io/badge/License-MIT-blue.svg" alt="License-MIT">
+    </a>{% endif %}
+    <a href="https://docs.rs/{{project-name}}">
+      <img src="https://img.shields.io/static/v1?label=Docs&message=docs.rs&color=blue" alt="Docs">
+    </a>{% if have_discord %}
+    <a href="{{discordlink}}">
+      <img src="https://img.shields.io/static/v1?label=Discord&message=join%20us!&color=mediumslateblue" alt="Discord">
+    </a>{% endif %}
+  </p>
+</div>
+
+<div align="center"><sub>:warning: Work in progress :warning:</sub></div>
+
+##
+
+## Outline
+
+- [Running the Webserver](#running-the-webserver)
+- [Testing the Project](#testing-the-project){% if bench %}
+- [Benchmarking the Project](#benchmarking-the-project){% endif %}{% if docker %}
+- [Running {{project-name}} on Docker](#running-{{project-name}}-on-docker){% endif %}{% if contributing %}
+- [Contributing](#contributing){% endif %}
+- [Getting Help](#getting-help)
+- [External Resources](#external-resources)
+- [License](#license)
+
+## Running the Webserver
+{% if nix == false %}
+First, due to upstream [Opentelemetry][otel] [dependencies][otel-issue], make
+sure to install the [protobuf][protobuf] compiler `protoc`.
+Instructions for various platforms can be found [here][protobuf-install].
+{% endif %}
+To start-up the [axum][axum] webserver, just run:
+
+```console
+cargo run
+```
+
+This will start-up the service, running on 2 ports:
+
+* `{{port}}`: main `{{project-name}}` application, including `/healthcheck`, etc.
+* `{{metricsport}}`: `/metrics`
+
+Upon running the application locally, (by default) [OpenAPI][openapi]
+documentation is made available as a [swagger-ui][swagger]
```

Comment:
```suggestion
Upon running the application locally, [OpenAPI][openapi]
documentation is available as a [swagger-ui][swagger]
```

Is there a way to opt-out of this during template generation? If not, I think "by default" can be removed.